mkconcore.py

```diff
@@ -10,38 +10,38 @@
 GRAPHML_FILE = sys.argv[1]
 TRIMMED_LOGS = True
 CONCOREPATH = "."
-CPPWIN    = "g++"        #Windows C++  6/22/21
-CPPEXE    = "g++"        #Ubuntu/macOS C++  6/22/21
-VWIN      = "iverilog"   #Windows verilog  6/25/21
-VEXE      = "iverilog"   #Ubuntu/macOS verilog  6/25/21
-CPPEXE    = "g++"        #Ubuntu/macOS C++  6/22/21
+CPPWIN    = "g++"        #Windows C++  6/22/21

```

Comment:
This file is modified by just line breaks changes, it seems?

Can you please make sure to commit only the actual changes, and not line breaks? Adding these changes makes it hard to track the development and makes the review difficult.

Please check with "git diff" before the pull request, and make sure such changes are not committed.